hoot-core/src/main/cpp/hoot/core/elements/ElementData.h

```diff
@@ -56,12 +56,14 @@ class ElementData
   static QString USER_EMPTY;
   static long UID_EMPTY;
   static bool VISIBLE_EMPTY;
+  static double CIRCULAR_ERROR_EMPTY;
+  static double CIRCULAR_ERROR_DEFAULT;
 
   virtual ~ElementData() {}
 
   virtual void clear() { _tags.clear(); }
 
-  Meters getCircularError() const { return _circularError >= 0 ? _circularError : 15.0; }
+  Meters getCircularError() const { return _circularError >= 0 ? _circularError : CIRCULAR_ERROR_DEFAULT; }
```

Comment:
This is the other big change.  This gets the "default" circular error from the configuration if one isn't set in the input data.